zebra/zebra_vty.c

```diff
@@ -2332,6 +2508,7 @@ DEFPY(ipv6_route_blackhole,
             |(1-255)$distance                              \
             |vrf NAME                                      \
             |label WORD                                    \
+	    |table (1-4294967295)                          \
```

Comment:
This needs to be a separate defun so that it can only be included when namespaces are configured.

Reply:
I had done it for vrf node commands, but was thinking that it was ok for default vrf. so I updated it.

Reply:
please fix the indentation as well